examples/widgets/pageable_list.dart

```diff
@@ -76,6 +77,13 @@ class PageableListApp extends App {
     return EventDisposition.processed;
   }
 
+  EventDisposition toggleItemsWrap() {
+    setState(() {
+      itemsWrap = !itemsWrap;
+    });
+    return EventDisposition.processed;
+  }
```

Comment:
You don't need this EventDisposition silliness anymore.  You can just have a void return value.